package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "processo-seletivo-backend-2024",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon ./src/server.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "nodemon": "^3.1.7"
+  },
+  "dependencies": {
+    "express": "^4.21.0",
```

geralmente fixamos a versão das dependências para garantir que nenhum update quebre o código
```suggestion
    "express": "4.21.0",
```

---

Anotado. Commitei a mudança!